bin/start.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+bundle install
```

Comment:
This is already happening in the Docker file.  Why is it necessary again?

Reply:
I was getting a gem not found error without it... but maybe I just doing something not right.

Reply:
OK, so after removing it just now locally... the error I get without it is 

```
app_1   | Could not find rake-13.0.0 in any of the sources
app_1   | Run `bundle install` to install missing gems.
```

Reply:
Actually, it's not already happening in the dockerfile. And, it's happening here so that it gets called every time we start the container.